services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -1221,8 +1247,8 @@ Optional<String> getTopicForCurrentPushJob(
     if (latestKafkaTopic.isPresent()) {
       LOGGER.debug("Latest kafka topic for store: {} is {}", storeName, latestKafkaTopic.get());
 
-      final String latestKafkaTopicName = latestKafkaTopic.get();
-      if (!isTopicTruncated(latestKafkaTopicName)) {
+      final PubSubTopic latestKafkaTopicName = latestKafkaTopic.get();
```

Comment:
nit: Maybe change the variable name here to be "latestPubSubTopic(/Name)"? 

(Again just to point out but not required to change, not critical for this PR)

Reply:
Or maybe just topic, just get rid of Kafka in the variable

Reply:
This is related to using `PubSubTopic` for `isTopicTruncated` function. Let me see if I can change the string topic to `PubSubTopic` for that function.

Reply:
I decide to do it in the future change.